BibBuddy/app/src/main/res/values/strings.xml

```diff
@@ -178,6 +179,12 @@
         Bitte füge mindestens ein Regal und ein Buch hinzu, um den Export zu ermöglichen.
     </string>
 
+    <!-- Export: Empty Note List -->
+    <string name="empty_note_list">Keine Notizen</string>
```

Comment:
Maybe you can make it consistent how you write the empty_note_list and empty_shelf message. In shelf you write a sentence and with the notes you have a short message. It is up to you if you want to change it.